### ISSUE TITLE
Help needed

### ISSUE THREAD

**Issue by anitaa1990** [alexbirkett/QuietShare#4]
Hey, I tried to clone this project in my system and run it in my device. The build was successful and I installed it in two devices:
Moto G5 Plus
Lenovo YT3-850M

But when I transmit something from either of the devices, it is not received in the other device. The microphone permission is enabled by default in the sample app and I have not made any modifications in the app before running it.

Any help on what the issue could be?

**Comment by brian-armstrong** [comment]
If https://quiet.github.io/quiet-js works in your browser, you might want to try interop with that to see if it's one of the phones that isn't working. The text modes there correspond to the 'audible' and 'ultrasonic-experimental' profiles in QuietShare.

When you run audible modes, can you hear the phone sending?

**Comment by alexbirkett** [comment]
Thanks for providing support @brian-armstrong! I'm not sure why these devices are not working.

**Comment by anitaa1990** [comment]
Yup, I tried sending using audible and I hear the sound when clicking on Transmit button.  
And yeah the quiet.js is working when i try from the browser. Will check in a couple of other devices and let you know

**Comment by brian-armstrong** [comment]
Just to be clear, did you try between browser and either phone? Were any of those successful?

**Comment by anitaa1990** [comment]
So using this link you provided: https://quiet.github.io/quiet-js
I was able to send data from my moto g5 plus to the browser using audible. But it did not work using ultrasound for some reason. 

And when I try to pass data from browser to moto g5 plus, using audible or ultrasound, the data was not received.
